src/bin/cargo.rs

```diff
@@ -116,7 +116,7 @@ fn execute(flags: Flags, config: &Config) -> CliResult<Option<()>> {
         // help message.
         "" | "help" if flags.arg_args.is_empty() => {
             config.shell().set_verbose(true);
-            let args = &["cargo".to_string(), "-h".to_string()];
```

Comment:
Currently in Cargo it's more idiomatic to just remember that `.to_string()` always works as opposed to switching between one of `.to_owned()`, `.cloned()`, or `.to_string()` as appropriate, so could these be backed out for now?